code/modules/mob/living/simple_animal/friendly/dog.dm

```diff
@@ -711,3 +711,59 @@ GLOBAL_LIST_INIT(strippable_corgi_items, create_strippable_list(list(
 /mob/living/simple_animal/pet/dog/corgi/lisa/Life(delta_time = SSMOBS_DT, times_fired)
 	. = ..()
 	make_babies()
+
+/mob/living/simple_animal/pet/dog/breaddog //Most of the code originates from Cak
+	name = "Sanallite"
+	desc = "It is a dog made out of bread. 'The universe is definitely half full'."
+	icon_state = "breaddog"
+	icon_living = "breaddog"
+	icon_dead = "breaddog_dead"
+	head_icon = 'icons/mob/pets_held_head.dmi'
+	health = 50
+	maxHealth = 50
+	gender = NEUTER
+	harm_intent_damage = 10
+	butcher_results = list(/obj/item/organ/internal/brain = 1, /obj/item/organ/internal/heart = 1, /obj/item/food/breadslice = 3,  \
+	/obj/item/food/meat/slab = 2)
+	response_harm_continuous = "takes a bite out of"
+	response_harm_simple = "take a bite out of"
+	attacked_sound = 'sound/items/eatfood.ogg'
+	deathmessage = "loses it's false life and collapses!"
+	deathsound = SFX_BODYFALL
+	held_state = "breaddog"
+	worn_slot_flags = ITEM_SLOT_HEAD
+
+/mob/living/simple_animal/pet/dog/breaddog/add_cell_sample()
+	return
+
+/mob/living/simple_animal/pet/dog/breaddog/CheckParts(list/parts)
+	..()
+	var/obj/item/organ/internal/brain/B = locate(/obj/item/organ/internal/brain) in contents
```

Comment:
wouldn't this be in the parts list? or is that not what it's for
also actual var name please, B means nothing